tests/align_parallel/test_align_wavefront_alignment_scheduler.h

```diff
@@ -200,7 +200,6 @@ SEQAN_DEFINE_TEST(test_align_parallel_wavefront_alignment_scheduler_async_with_e
         test_align_parallel::RaiiEvent event;
         TInnerTask task = [&]()
         {
-            if (calledIds[id] > 0)
```

Comment:
is this change intended?

Reply:
yes, because it solves the race condition seen in the PPC test.